src/components/ComboBox/ComboBox.jsx

```diff
@@ -71,52 +69,52 @@ const ComboBox = ({
     // Only apply logic if user hit enter
     if (evt.key === 'Enter' && currentValue !== '' && currentValue !== undefined) {
       // Check if there is already a tag for this value and return it
-      const filteredTags = tagItems.filter(x => itemToString(x) === currentValue);
+      const hasNoFilteredTag = tagItems.filter(x => itemToString(x) === currentValue).length < 1;
```

Comment:
I think the comment above here, and possibly some other comments can be removed now or at least modified